cryptofeed/exchange/dsx.py

```diff
@@ -0,0 +1,114 @@
+'''
+Copyright (C) 2017-2020  Bryant Moscon - bmoscon@gmail.com
+
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+import json
+import logging
+from decimal import Decimal
+
+from sortedcontainers import SortedDict as sd
+
+from cryptofeed.feed import Feed
+from cryptofeed.defines import TICKER, L2_BOOK, TRADES, BUY, SELL, BID, ASK, DSX
```

Comment:
probably should import DSX here as DSX_id, or _DSX, etc.

```python
from cryptofeed.defines import TICKER, L2_BOOK, TRADES, BUY, SELL, BID, ASK, DSX as DSX_id
```

then

```python
class DSX(Feed):
    id = DSX_id
```

Reply:
corrections made and pushed